src/util/has_own.ts

```diff
@@ -0,0 +1,5 @@
+import {getOwn} from './get_own';
+
+export function hasOwn<T extends object>(object: T, key: PropertyKey): key is keyof T {
```

Comment:
I think this is redundant, isn't it? 
Is this needed for strict value checking?

Reply:
`getOwn()` alone probably serves all use cases right now. Get rid of it?

Reply:
Sure.

Reply:
Resolved in d8fdb0d7ff2971a9da5212062a2161c1e93f3ac1